lib/potassium/recipes/api.rb

```diff
@@ -31,4 +43,67 @@ def add_api
       generate "power_api:install"
     end
   end
+
+  def add_graphql
+    gather_gem 'graphql'
+    if get(:authentication)
+      gather_gem 'jwt'
+    end
+    gather_gems(:development, :test) do
+      gather_gem 'graphql_playground-rails'
+    end
+
+    after(:gem_install) do
+      generate "graphql:install --skip_graphiql"
+      playground_route = <<~HEREDOC
+      \n
+        if Rails.env.development?
+          mount GraphqlPlayground::Rails::Engine, at: "/graphiql", graphql_path: "/graphql"
+        end
+      HEREDOC
+      inject_into_file('config/routes.rb',
+        playground_route,
+        after: 'post "/graphql", to: "graphql#execute"'
+      )
+      copy_file(
+        "../assets/config/graphql_playground.rb",
+        "config/initializers/graphql_playground.rb"
+      )
+      remove_dir 'app/graphql/types'
+      directory '../assets/app/graphql/types', 'app/graphql/types'
+      gsub_file 'app/graphql/mutations/base_mutation.rb', 'Types::Base', 'Types::Base::Base'
+      directory '../assets/app/graphql/queries', 'app/graphql/queries'
+      gsub_file 'app/graphql/mutations/base_mutation.rb', 'RelayClassic', ''
+      gsub_file 'app/graphql/mutations/base_mutation.rb', "    input_object_class Types::Base::BaseInputObject\n", ''
+
+      if get(:authentication)
+        copy_file(
+          '../assets/app/graphql/graphql_controller.rb',
+          'app/controllers/graphql_controller.rb',
+          force: true
+        )
+        gsub_file(
+          'app/controllers/graphql_controller.rb',
+          'GqlSampleSchema',
+          "#{get(:titleized_app_name).delete(' ')}Schema"
+        )
+        copy_file(
+          '../assets/app/graphql/mutations/login_mutation.rb',
+          'app/graphql/mutations/login_mutation.rb'
+        )
+        inject_into_file('app/graphql/types/mutation_type.rb',
+          "\n    field :login, mutation: Mutations::LoginMutation",
+          after: 'class MutationType < Types::Base::BaseObject'
+        )
```

Comment:
Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.